.changesets/fix_bnjjj_fix_coprocessor_with_subs.md

```diff
@@ -0,0 +1,5 @@
+### make coprocessors work with subscriptions ([PR #5542](https://github.com/apollographql/router/pull/5542))
+
+Do not override skipped data from serde when you returns a body from a coprocessor response stage.
```

Comment:
cc @bnjjj , not sure this level of implementation detail is needed, is this enough:

```suggestion
### Enable coprocessors for subscriptions ([PR #5542](https://github.com/apollographql/router/pull/5542))

Ensures that coprocessors correctly handle subscriptions by preventing skipped data from being overwritten.
```